src/TrackRowInfoVisQuantitativeBar.js

```diff
@@ -143,22 +145,30 @@ export default function TrackRowInfoVisQuantitativeBar(props) {
                     }
                 });
 
+                if(isAggregated) {
+                    const aggIndicatorWidth = 2;
```

Comment:
Maybe instead of these indicator bars over the existing visualizations, we could add an extra `TrackRowInfoVisQuantitativeBar` visualization that shows how many samples are grouped in each visible row.

So the bars would all have width `1` when there is no aggregation, and different heights for the size of each group when there is aggregation.

Reply:
I supported additional aggregation functions including `count` and `countUnique`.

You can use `count` to show the number of samples aggregated in each row using `TrackRowInfoVisQuantitativeBar` visualization, just like a track on the left-most side of this figure: 
![Screen Shot 2020-06-05 at 2 37 48 PM](https://user-images.githubusercontent.com/9922882/83911376-2641a700-a73a-11ea-9910-3a15bc0fb677.png)

I removed the vertical indicators. I think this kind of 'compact' indicators would be still necessary, but I guess these vertical indicators positioned on the every visualizations are a bit distracting and confusing, especially when one aggregate all rows (`oneOf` === undefined).